### DISCUSSION TITLE
Pest is not generating the Rule on windows

### DISCUSSION THREAD

**Discussion by victorinno** [pest-parser/pest#626] [General]
Hi there, I might be missing something but when I execute `cargo doc` ou `cargo build` the Rule is not generated.

my cargo.toml:
```[package]
name = "accounter"
version = "0.1.0"
edition = "2018"

[dependencies]
pest = "2.1"
pest_derive = "2.1"
anyhow = "1.0"
rustyline = "6.2"
cfg-if = "0.1"
lazy_static = "1.0"
```

The grammar I get from the book example:
```
num = @{ int ~ ("." ~ ASCII_DIGIT*)? ~ (^"e" ~ int)? }
    int = { ("+" | "-")? ~ ASCII_DIGIT+ }

operation = _{ add | subtract | multiply | divide | power }
    add      = { "+" }
    subtract = { "-" }
    multiply = { "*" }
    divide   = { "/" }
    power    = { "^" }

expr = { term ~ (operation ~ term)* }
term = _{ num | "(" ~ expr ~ ")" }

calculation = _{ SOI ~ expr ~ EOI }

WHITESPACE = _{ " " | "\t" }

```

**Comment by rob-mur** [comment]
Hi!

To try and help you, I started by reproducing the error. See here: [https://github.com/rob-mur/pest-bug-repr](url)

(Note that in future it would be helpful if you had already done this stage)

Upon writing a simple test to parse "1 + 2" and print the result it seems to work fine for me. Could you please take a look and see where yours is different?

(Edit: changed from a+b to 1+2 as the calculator example is only for numbers)

**Comment by tomtau** [comment]
if it's a linker issue, you can see if this helps: https://github.com/pest-parser/pest/discussions/622#discussioncomment-3114674
